firestore/FirestoreSwiftUIExample/ViewModels/RestaurantViewModel.swift

```diff
@@ -60,9 +103,17 @@ class RestaurantViewModel: ObservableObject {
             return nil
           }
         }
-
-        print(self.reviews)
       }
     }
   }
+
+  func getNSError(document: DocumentSnapshot) -> NSError {
+    return NSError(
+      domain: "AppErrorDomain",
+      code: -1,
+      userInfo: [
+        NSLocalizedDescriptionKey: "Unable to retrieve population from snapshot \(document)"
+      ]
+    )
+  }
```

Comment:
Swift lets you declare functions with almost the same level of freedom as JavaScript, so you could copy+paste this whole method into line 54 since it's not used anywhere else (and has no dependency on `self`).

Reply:
I put this in a function since it's used in two places, just for code cleanliness.